de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SessionUtils.java

```diff
@@ -0,0 +1,82 @@
+package de.fu_berlin.inf.dpp.session;
+
+import de.fu_berlin.inf.dpp.SarosPluginContext;
+import de.fu_berlin.inf.dpp.activities.SPath;
+import de.fu_berlin.inf.dpp.filesystem.IResource;
+
+import org.picocontainer.annotations.Inject;
+
+/**
+ * A utility class to provide easy access to functionality needing a session
+ * object.
+ */
+public class SessionUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=293&issues=AWbRiJzSSgN4uCclDZzy&open=AWbRiJzSSgN4uCclDZzy)

[ISSUE_KEY]: AWbRiJzSSgN4uCclDZzy